### PR TITLE
docs: add clarifications to readme

### PR DESCRIPTION
**Pull request by benmccann**
- show how to set it up without the recommended configs. I only wanted to setup a single rule and had a very difficult time
- show the new config format first and the legacy format after
- don't duplicate legend symbols

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,35 +26,69 @@ npm install --save-dev eslint eslint-plugin-n
...
+To setup without the recommended configs, you'll need to add the plugin:
 
 ```js
 const nodePlugin = require("eslint-plugin-n")
 
 module.exports = [
-    nodePlugin.configs["flat/recommended-script"],
     {
+		plugins: {
+			n: node
```
Comment (scagood) [inline README.md:52]:
```suggestion
			n: nodePlugin
```

File: README.md [hunk 2]
```diff
@@ -26,35 +26,69 @@ npm install --save-dev eslint eslint-plugin-n
...
+To setup without the recommended rules you'll need to add the plugin:
+
+```jsonc
+{
+    "parserOptions": {
+        "ecmaVersion": 2021
+    },
+    "plugins": {
+		"n": node
+    },
```
Comment (scagood) [inline README.md:84]: Im not sure this'll work in the legacy version?

**Review by scagood — CHANGES_REQUESTED** [review]
I generally like the docs changes, I have added a couple of small points.

Thank you :)

File: README.md (hunk 2 above)
Comment (benmccann) [reply README.md:84]: I don't know. I only have the new version to test on. We can remove this section

File: README.md [hunk 3]
```diff
@@ -26,35 +26,69 @@ npm install --save-dev eslint eslint-plugin-n
...
```
Comment (aladdin-add) [inline README.md:84]: in eslintrc mode, it should be:
```suggestion
    "plugins": ["n"],
```

**Comment by aladdin-add** [comment]
can you look at the ci failing - seems markdown-lint was not happy somehow: https://github.com/eslint-community/eslint-plugin-n/actions/runs/9725270531/job/26842381101?pr=303

**Comment by aladdin-add** [comment]
it has been landed in dbdfa8e0abdeb96f2f843c4112cc43f16f2657ea. thanks for contributing!:)

**Comment by aladdin-add** [comment]
it has been landed in dbdfa8e0abdeb96f2f843c4112cc43f16f2657ea. thanks for contributing!:)

**Pull request closed without merging**